CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## [0.11]
+
+- **Refactored** Renderers - the `makeRenderer` functions in `server-render`
```

Comment:
more questions :)

So a for instance use of this might be, that it could be used if we want the application to server all url's under `/admin`. Would this mean the routes would _not_ need to be prefixed with `/admin` in the route path setup? (e.g. const `route = { path: 'dashboard' }` vs `const route = { path: 'admin/dashboard' }`

i'm not 100% sure what is meant by "Corresponding changes in the signatures to a few application-related modules mean that if you're relying on direct imports of those modules, your code will need to be updated."